wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1624,13 +1624,20 @@ class KotlinGenerator private constructor(
     return CodeBlock.of("%T.%L", type.typeName, enumConstant.name)
   }
 
-  private fun Field.typeNameForBuilderSetter(baseClass: TypeName = type!!.asTypeName()): TypeName {
+  private fun Field.typeNameForBuilderSetter(): TypeName {
```

Comment:
is there logic to share with the property declaration?

Reply:
Not yet!